src/commands/hexpireat.json

```
@@ -0,0 +1,120 @@
+{
+    "HEXPIREAT": {
+        "summary": "Set expiry time on hash fields.",
+        "complexity": "O(1) for each field assigned with TTL, so O(N) to add N TTLs when the command is called with multiple fields.",
+        "group": "hash",
+        "since": "9.0.0",
+        "arity": -6,
+        "function": "hexpireAtCommand",
```

nit - match `expireat`?
```suggestion
        "function": "hexpireatCommand",
```

---

O.K. but I find it more clear using the camelcase